Cargo.toml

```diff
@@ -8,6 +8,7 @@ homepage = "https://github.com/gfx-rs/naga"
 repository = "https://github.com/gfx-rs/naga"
 keywords = ["shader", "SPIR-V", "GLSL", "MSL"]
 license = "MIT OR Apache-2.0"
+resolver = "2"
```

Comment:
A new feature resolver has been introduced in [rust 1.51](https://blog.rust-lang.org/2021/03/25/Rust-1.51.0.html). Naga MSRV is 1.43 (based on info from CI task, but this definitely should be in README).

Is it required here?

Reply:
It is required if the naga cli uses features, because otherwise `cargo run -p naga-cli --features ...` does not work: https://github.com/rust-lang/cargo/issues/5364.

Is the MSRV just for _users_ of `naga` or also for users of the `naga` binary and naga developers? The `resolver = 2` only affects the latter, older cargo version can still use naga as a dependency.

Reply:
Thank you for explain! MSRV for `naga` lib users. Everything is ok, since CI is happy.